utils/install-python.bat

```diff
@@ -0,0 +1,44 @@
+@echo off
+setlocal enabledelayedexpansion
+
+:: Define the Python installer URL and installation directory
+set "pythonInstallerURL=https://www.python.org/ftp/python/3.11.5/python-3.11.5.exe"
+set "installDir=C:\Python"  :: Replace with your desired installation directory
```

Comment:
Could we maybe install a python into the extension's path? You can modify `~/.vscode/extensions/...` after having it installed. That would be neater IMHO.

Or we just pre-package a windows python and use it on windows; and just assume other platforms are sane.

Reply:
https://www.python.org/ftp/python/3.11.6/python-3.11.6-embed-arm64.zip seems to be what we want...